services/client/src/components/EventLogsTable.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import TableList from 'components/TableList';
+
+const EventLogsTable = ({ events }) => {
+  const columns = [
+    {
+      title: 'Event',
+      dataIndex: 'event',
+      key: 'event',
+    },
+    {
+      title: 'Duration (ms)',
+      dataIndex: 'duration',
+      key: 'duration',
+      render: (_, __, i) => {
+        const sortedEvents = events?.sort((a, b) => a.timestamp > b.timestamp);
```

Comment:
right now it's calculating for every row in the table, should be calculated once